usaspending_api/common/helpers/date_helper.py

```diff
@@ -1,11 +1,16 @@
 import operator
 
 from argparse import ArgumentTypeError
-from datetime import timezone, datetime
+from datetime import datetime, timezone
```

Comment:
I'm gonna miss you Kirk

Reply:
lol... nah... you're just gonna miss my OCDs.